src/window/mod.rs

```diff
@@ -147,6 +147,9 @@ pub fn create_window(event_loop: &ActiveEventLoop, maximized: bool, title: &str)
         .with_transparent(true)
         .with_visible(false);
 
+    #[cfg(target_os = "windows")]
+    let window_attributes = window_attributes.with_taskbar_icon(Some(load_icon()));
```

Comment:
I think we could avoid loading the icon twice since it is loaded at [L132](https://github.com/neovide/neovide/pull/2816/files#diff-3236d45aa6e68d88a718f7a9e523800f3a3da86bfb4114a11c320382149b4b24R132)

to achieve this we need to .clone() the icon only for windows specifically, so I would suggest something like this for readability would be enough:

```rust
let window_attributes = Window::default_attributes()
    .with_title(title)
    .with_maximized(maximized)
    .with_transparent(true)
    .with_visible(false);

#[cfg(target_family = "unix")]
let window_attributes = window_attributes.with_window_icon(Some(icon));

#[cfg(target_os = "windows")]
let window_attributes = window_attributes
    .with_window_icon(Some(icon.clone()))
    .with_taskbar_icon(Some(icon));
```